CHANGES.md

```diff
@@ -2,6 +2,7 @@
 
 ## 0.2 - 2024-04-24
 
+- add `owi conc` subcommands and `owi c --concolic`: concolic mode
```

Comment:
This should go in the `unreleased` section, 0.2 has already been published on opam :)